browser-test/src/support/ti_dashboard.ts

```diff
@@ -0,0 +1,54 @@
+import {Page} from 'playwright'
+
+export class TiDashboard {
```

Comment:
Add comment explaining that this class helps to interfact with Trusted intermediatery dashboard page. And requires that user logs in as TI first.